src/wallet/wallet.cpp

```diff
@@ -2115,7 +2115,7 @@ bool CWallet::GetNewDestination(const OutputType type, const std::string label,
         spk_man->TopUp();
         result = spk_man->GetNewDestination(type, dest, error);
     } else {
-        error = strprintf("Error: No %s addresses available.", FormatOutputType(type));
+        error = strprintf(_("Error: No %s addresses available."), FormatOutputType(type)).translated;
```

Comment:
Not sure how difficult it would be to fix, but the general idea for translations is to pass up the bilingual string and let the caller pick the right version. Otherwise the RPC responses (and debug log) might be partially translated. Functional test might fail when run with the gui.

Reply:
I think that will be a bit invasive since several of the functions involved use `std::string error` rather than `bilingual_str`. Perhaps for a followup.

Reply:
Yes, if you're going to translate anything, it's better to use `bilingual_str`. I would prefer to do this right in the first go, instead of having another follow-up commit to prevent RPC messages from being translated.

Alternatively just leave this untranslated for now. This is marked for 22.0 and we've passed the translation string freeze.

Reply:
For the strings I added, I was just following the pattern already in use in these functions. For this change in particular, it was to make the style unified. Unless this is considered a blocker, I'm going to leave it as is.

Reply:
Ok…